android/src/org/kiwix/kiwixmobile/library/entity/LibraryNetworkEntity.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2016
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU  General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301, USA.
+ */
+package org.kiwix.kiwixmobile.library.entity;
+
+import java.util.List;
+import org.simpleframework.xml.Attribute;
+import org.simpleframework.xml.ElementList;
+import org.simpleframework.xml.Root;
+
+@Root(name = "library")
+public class LibraryNetworkEntity {
```

Comment:
It's a bean model of a the kiwix.org/library.xml. I have not yet committed the glue code that does the actual parsing - about to commit that rn :)